docs/DeveloperGuide.md

```diff
@@ -141,12 +141,12 @@ The `Model` component,
 
 The `Storage` component,
 * can save both address book data and user preference data in json format, and read them back into corresponding objects.
-* inherits from both `AddressBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
+* inherits from both `UniNurseStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
```

Comment:
A recurring issue: `UniNurseStorage` should be `UninurseBookStorage`
```suggestion
* inherits from both `UninurseBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
```

Reply:
Updated